v1/types/types.go

```diff
@@ -17,6 +17,22 @@ import (
 	"github.com/open-policy-agent/opa/v1/util"
 )
 
+var (
+	// Nl represents an instance of the null type.
+	Nl Type = NewNull()
```

Comment:
Interesting choice. Leaving some ambiguity between Null and Nil. And the Netherlands. Is there a reason we cannot use "Null" here?